core-bundle/src/Resources/contao/controllers/FrontendShare.php

```diff
@@ -26,19 +26,29 @@ class FrontendShare extends Frontend
 	 */
 	public function run()
 	{
+		if (($url = Input::get('u', true)) && !\is_string($url))
+		{
+			return new RedirectResponse('../');
```

Comment:
can you please explain what this redirect is supposed to do? Maybe add a comment?

Reply:
This is an early return. The redirect already exists at the end of the method. :)

Reply:
still a very strange response, isn't it? Should it point to the root instead?

Reply:
Do you mean `new RedirectResponse('/');`?